MODULE.bazel

```
@@ -0,0 +1,64 @@
+"""remote-apis dependencies"""
+
+module(
+    name = "bazel_remote_apis",
```

```suggestion
    name = "remoteapis",
```

bazel itself uses this name, not sure which should be preferred